packages/vulcan-core/lib/modules/components/Datatable/DatatableFilter.jsx

```diff
@@ -62,7 +62,7 @@ const DatatableFilter = props => {
           />
         }
         size="small"
-        trigger={<Filter count={getCount(columnFilters)} />}>
+        trigger={Filter({ count: getCount(columnFilters) })}>
```

Comment:
I don't get this change sorry. Why would you need to call the React component directly as a function? That could provoke performance issuses

Reply:
More precisely what's the behaviour/error you encounter when directly using a React component?

Reply:
Il was the only way for the trigger to accept the onClick to open the modal.